client/blocks/taxonomy-manager/list-item.jsx

```diff
@@ -19,11 +19,28 @@ import Dialog from 'components/dialog';
 import { getSelectedSiteId } from 'state/ui/selectors';
 import { getSiteSettings } from 'state/site-settings/selectors';
 import { getSite, isJetpackSite } from 'state/sites/selectors';
+import { decodeEntities } from 'lib/formatting';
 import { deleteTerm } from 'state/terms/actions';
 import { saveSiteSettings } from 'state/site-settings/actions';
-import { decodeEntities } from 'lib/formatting';
+import { setLayoutFocus } from 'state/ui/layout-focus/actions';
+import { setPreviewUrl, setPreviewType } from 'state/ui/preview/actions';
+import { setUrlScheme } from 'lib/url';
 import Tooltip from 'components/tooltip';
 
+const ViewTaxonomyMenuItem = ( { href, onClick, isPreviewable, children } ) => {
```

Comment:
Shouldn't we extract this component to a more reusable place? It seems that this behaviour could be shared between different places and not only taxonomies.